app/controllers/api/student_submissions_controller.rb

```diff
@@ -79,6 +79,8 @@ def student_submission_params
       :guardian_two_phone_number,
       :guardian_two_phone_type,
       :guardian_two_relationship,
+      :health_conditions,
```

Comment:
There's already a declaration at the bottom ~ we have to declare `array` type params after everything else.